README.md

```diff
@@ -7,10 +7,19 @@ Knife Windows Plugin
 This plugin adds additional functionality to the Chef Knife CLI tool for
 configuring / interacting with nodes running Microsoft Windows:
 
-* Bootstrap of nodes via the [Windows Remote Management (WinRM)](http://msdn.microsoft.com/en-us/library/aa384426\(v=VS.85\).aspx) or SSH protocols
 * Remote command execution using the WinRM protocol
 * Utilities to configure WinRM SSL endpoints on managed nodes
 
+## **Knife bootstrap windows ssh/winrm**
+
+Bootstrapping has been moved into chef-core and is no longer a part of knife-window
+
+| Old knife-windows | New knife (chef-core) | Notes |
+|-----:|:-----------|:-----------|
+| knife windows bootstrap ssh |knife bootstrap -o ssh or knife bootstrap | Default is 'ssh'. | |
+| knife windows bootstrap winrm | knife bootstrap -o winrm| | |
```

Comment:
nitpicks:
* there is some whitespace inconsistency here to fix.
* I would drop the period after `Default is 'ssh'`
* It seems like there's an unused fourth column here

Reply:
Thank you @btm for such pointers.

Reply:
s/(chef-core)/(train)

Reply:
Hey @marcparadise 
> s/(chef-core)/(train)

Could you please guide me on this? Thanks

Reply:
@vsingh-msys he meant to replace "chef-core" with "train" but that was covered by a different change. `s/foo/bar/` is a regular expression substitution and is often used by developers to quickly say "replace foo with bar".